openfisca_us/variables/demographic/person.py

```diff
@@ -43,3 +43,76 @@ def formula(person, period, parameters):
             [age < 18, age < 65, age >= 65],
             [AgeGroup.CHILD, AgeGroup.WORKING_AGE, AgeGroup.SENIOR],
         )
+
+
+class ccdf_age(Variable):
+    value_type = float
+    entity = Person
+    label = u"CCDF age"
+    definition_period = YEAR
+
+
+class CCDFCareLocation(Enum):
+    CENTER_BASED = "Center-based"
+    HOME_BASED = "Home-based"
+
+
+class ccdf_care_location(Variable):
+    value_type = Enum
+    possible_values = CCDFCareLocation
+    default_value = CCDFCareLocation.CENTER_BASED
+    entity = Person
+    label = u"CCDF care location"
+    definition_period = YEAR
```

Comment:
Is this a function of `provider_type`?

Reply:
Yeah actually they are
![image](https://user-images.githubusercontent.com/26745287/139598425-935e3f90-d0b5-401c-9f46-2514682861cb.png)

Reply:
DCC_SACC -> center-based
FDC_GFDC -> home-based
LE_STD -> home-based
LE_ENH -> home-based

Reply:
I'll need to change the formula then.

Reply:
My current branch, child_care_age_group, doesn't have the enums for provider_type. How should I approach with this?